### DISCUSSION TITLE
Error message problem since last ACE update

### DISCUSSION THREAD

**Discussion by korda2310** [Vdauphin/HeartsAndMinds#1662] [Question and answer]
It is impossible to launch the mission since this last ACE update. I have this error message.
![Capture](https://github.com/user-attachments/assets/6dd3d547-099a-4aa0-90eb-27f8d4bcad79)

**Comment by Vdauphin** [comment]
Hello,

Working from here
Ask on A3 Official Discord for help

Cheers

**Reply by korda2310** [reply]
Sorry for my English via Google Translate (I speak French), I meant since the ACE update (3.18.0) it is impossible to launch your mission "HeartsAndMinds" because of the error message, the one in the screenshot..With the other scenarios I don't have this problem except for your scenario only..

**Reply by Vdauphin** [reply]
It is still working for me
It should be an issue on your side, as said, ask in A3 Official Discord for help
